models/sipnet/R/write_restart.SIPNET.R

```diff
@@ -104,14 +104,14 @@ write_restart.SIPNET <- function(outdir, runid, start.time, stop.time, settings,
   }
   
   if ("SoilMoistFrac" %in% variables) {
-    analysis.save[[length(analysis.save) + 1]] <- new.state$SoilMoistFrac  ## unitless
-    if (new.state$SoilMoistFrac < 0 || new.state$SoilMoistFrac > 1) analysis.save[[length(analysis.save)]] <- 0.5
+    analysis.save[[length(analysis.save) + 1]] <- new.state$SoilMoistFrac/100  ## unitless
+    if (analysis.save[[length(analysis.save)]] < 0 || analysis.save[[length(analysis.save)]] > 1) analysis.save[[length(analysis.save)]] <- 0.5
```

Comment:
not following the change in logic here

Reply:
So if we use the percentage of Soil moisture profile (0-100) and compare the percentage value against the 0 - 1 range, the soilMoistFrac will not be updated correctly and will be fixed at 0.5 in most cases.